certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+            "description": "Output format"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
             "schema": {
-              "type": "object",
-              "properties": {
-                "input": {
-                  "type": "string",
-                  "description": "The string to search for a match",
-                  "title": "Text to Match"
-                },
-                "pattern": {
-                  "type": "string",
-                  "description": "The .NET regular expression pattern to match",
-                  "title": "Regular Expression"
-                },
-                "option": {
-                  "x-ms-summary": "Regular expression option",
-                  "x-ms-dynamic-values": {
-                    "operationId": "GetRegexOptions",
-                    "value-path": "name",
-                    "value-title": "name"
-                  },
-                  "description": "Select regular expression option",
-                  "type": "string",
-                  "default": "IgnoreCase"
-                }
-              },
-              "required": [
-                "pattern",
-                "input",
-                "option"
-              ]
-            }
-          }
-        ]
+              "$ref": "#/definitions/FileResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Create QR or Barcode",
+        "description": "Create a QR or Barcode from an string"
+      }
+    }
+  },
+  "definitions": {
+    "CodeResponse": {
+      "type": "object",
+      "properties": {
+        "codeValue": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Code value",
+          "description": "Code value"
+        },
+        "codeType": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Code type",
+          "description": "Code type"
+        }
+      }
+    },
+    "CsvData": {
+      "required": [
+        "csv"
+      ],
+      "type": "object",
+      "properties": {
+        "csv": {
+          "minLength": 1,
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "CSV",
+          "description": "CSV"
+        }
+      }
+    },
+    "CsvResponse": {
+      "type": "object",
+      "properties": {
+        "csv": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "CSV response",
+          "description": "CSV response"
+        }
+      }
+    },
+    "FileContentResponse": {
+      "type": "object",
+      "properties": {
+        "fileContent": {
+          "x-ms-visibility": "important",
+          "x-ms-summary": "File response",
+          "description": "File response"
+        }
+      }
+    },
+    "FileData": {
+      "required": [
+        "file"
```

Comment:
@stsiek The required input field has been added. Existing flows will not send this new field.